platform/android/MapboxGLAndroidSDK/.gitignore

```diff
@@ -0,0 +1,2 @@
+lint-baseline.xml
+lint/lint-baseline-local.xml
```

Comment:
@Guardiola31337 Thanks for the explanation 👍 . Could we add a big note to the XML lint files saying that? Something like "remember to update file XXXX.XML as well if you make any changes to this file".